### PR TITLE
Feature/home page component

### PR DESCRIPTION
**Pull request by KarrarQ**
# Description

As a user, when I load the app, I should see a landing page 
  - Created buttons with links to other routes.
   -Added menu items
-
Fixes # (issue)

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Styling update

# How Has This Been Tested?

- [X] Manual Testing
- [ ] Cypress Testing

# Checklist:

- [X] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [X] Lint generates no new warnings
- [X] I have checked my terminal and there are no errors
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have checked that all cypress tests are still passing

# Comments for Reviewer(s)

-

### CODE REVIEW

**Review by Tristinsorrells1 — APPROVED** [review]
Looks good!

File: src/Components/App/App.css
Comment (BHaddad1) [inline src/Components/App/App.css:1]: Looks good!

File: src/Components/App/App.tsx
Comment (BHaddad1) [inline src/Components/App/App.tsx:1]: This also seems fine!

File: src/Components/Home/Home.css
Comment (BHaddad1) [inline src/Components/Home/Home.css:1]: Looks like a lot of CSS, thanks!

**Review by BHaddad1 — APPROVED** [review]
These changes look good, thanks for working on this!

**Pull request merged**